.gitignore

```diff
@@ -1,2 +1,6 @@
 myvenv
-*.pyc
\ No newline at end of file
+*.pyc
+storagemanager/storage/__pycache__/
+storagemanager/storage/migrations/__pycache__/
+storagemanager/storagemanager/__pycache__/
```

Comment:
You'll have to add pycache each time for new folder, will it work with i.e. `*/__pycache__/*`?